packaging/cmake/Modules/Packaging.cmake

```diff
@@ -9,6 +9,8 @@ set(CPACK_DEBIAN_DEBUGINFO_PACKAGE NO)
 
 set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS YES)
 
+set(PKG_FILES "${CMAKE_SOURCE_DIR}/packaging/cmake/pkg-files")
```

Comment:
This is not really needed, is it? I think can use a single definition from the main CMakeLists.txt file.

Reply:
It probably isn’t needed, I just kind of forgot it was there. In the the initial version of this change I had forgotten to modify the main CMakeLists file, and when I fixed that I forgot to remove this line in the packaging file. I’ll update the PR to exclude it.